week8/UniversityEntryView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UniversityEntryView.swift
+//  Mod8JSON
+//
+//  Created by Nick Ryan on 6/8/24.
+//
+
+import SwiftUI
+
+struct UniversityEntryView: View {
+  let university: UniversityEntry
+  
+  var body: some View {
+    NavigationStack {
```

Comment:
You don't need this navigation stack here. This view is already pushed inside a navigation stack.